### ISSUE TITLE
[tune]One of example is broken

### ISSUE THREAD

**Issue by MarkWh1te** [ray-project/ray#5575]
### System information
- Linux Ubuntu 18.04
- **Ray installed from (pip )**:
- **Ray version**:
ray                           0.7.3     
- **Python version**:
Python 3.6.5
- **Exact command to reproduce**:
https://github.com/ray-project/ray/blob/master/python/ray/tune/examples/pbt_ppo_example.py

### Describe the problem
I cannot run this file.
The compiler shows these lines:
Traceback (most recent call last):
  File "pbt_ppo_example.py", line 73, in <module>
    lambda spec: random.choice([10000, 20000, 40000]))
TypeError: run() got an unexpected keyword argument 'env'
### Source code / logs
https://github.com/ray-project/ray/blob/master/python/ray/tune/examples/pbt_ppo_example.py

**Comment by richardliaw** [comment]
Can you try:

```diff
         name="pbt_humanoid_test",
         scheduler=pbt,
         **{
-            "env": "Humanoid-v1",
             "num_samples": 8,
             "config": {
+                "env": "Humanoid-v1",
                 "kl_coeff": 1.0,
                 "num_workers": 8,
                 "num_gpus": 1,
```

**Comment by richardliaw** [comment]
Should be fixed in #5558
